siriuspy/siriuspy/optics/lattice_survey.py

```diff
@@ -60,6 +60,11 @@ def _append_mag_data(filename, model, acc, label, section):
     mag_tps.extend(model.families.families_vertical_correctors())
     mag_tps.extend(model.families.families_sextupoles())
     mag_tps.extend(model.families.families_skew_correctors())
+    try:
```

Comment:
I prefer this style of coding:
```python3
if acc.lower().startswith('si'):
    mag_tps.extend(model.families.families_id_correctors()
```
It is shorter and self describable.

Reply:
ok, done.